src/services/index.js

```diff
@@ -18,8 +19,9 @@ module.exports = function () { // eslint-disable-line func-names
   mongoose.connect(app.get('mongodb'));
   mongoose.Promise = Promise;
 
-  app.configure(authentication);
+  app.configure(activityLog);
   app.configure(app3rd);
+  app.configure(authentication);
```

Comment:
Have you checked that there is no effect if moving authentication below other services?